pkg/clusteragent/clusterchecks/dispatcher_endpoints_configs.go

```diff
@@ -61,6 +62,12 @@ func (d *dispatcher) patchEndpointsConfiguration(in integration.Config) (integra
 	out := in
 	out.ClusterCheck = false
 
+	if out.Provider == names.CloudFoundryBBS {
```

Comment:
should we patch cluster checks as well? correct if I'm wrong, but I believe we want to run classic cluster checks on CF not only endpoint checks

Reply:
already done here it seems http://github.com/Datadog/datadog-agent/blob/8c0de18e12777494856ac3825a6bb0c97ee37cc5/pkg/clusteragent/clusterchecks/dispatcher_configs.go#L137-L138